core/server/master/src/test/java/alluxio/master/metrics/MetricsMasterTest.java

```diff
@@ -94,7 +94,7 @@ public void testAggregator() {
         Metric.from("worker.192_1_1_2.metricA", 3, MetricType.GAUGE));
     mMetricsMaster.workerHeartbeat("192_1_1_2", metrics3);
     assertEquals(13L, getGauge("metricA"));
-    assertEquals(20L, getGauge("metricB"));
+    assertEquals(22L, getGauge("metricB"));
```

Comment:
This is related to whether we want to keep the old metrics. For the previous implementation, if a metrics is not reported in the current heartbeat but reported in the previous heartbeats by a worker/client, it will be deleted.
In my implementation, even if a metric is not reported in the current heartbeat, we should also keep its old value to show the more correct metrics, especially for MetricType=VALUE.

I think the previous implementation helps remove old metrics reported by old clients, but doesn't make sense for worker metrics which is always stable and need to record old values(COUNTERS)